testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -167,7 +167,27 @@ def test_write_frame_none(self, u, tmpdir, extension):
             u.atoms.positions[None, ...], new_positions, decimal=2
         )
 
+    @pytest.mark.parametrize('extension', ('xtc', 'dcd', 'pdb', 'xyz'))
+    def test_compressed_write_frame_none(self, u, tmpdir, extension):
+        for ext in ('.gz', '.bz2'):
+            destination = str(tmpdir / 'test.' + extension + ext)
+            u.atoms.write(destination, frames=None)
+            u_new = mda.Universe(destination)
+            new_positions = np.stack([ts.positions for ts in u_new.trajectory])
+            assert_array_almost_equal(
+                u.atoms.positions[None, ...], new_positions, decimal=2
```

Comment:
`decimal=2` is a low bar, can't we increase the precision, at least for anything that is not XTC?

Reply:
(XTC is a lossy format, so XTC compared to original might indeed only be accurate to 2 decimals)

Reply:
I assume that you used what we already had in the tests... then ignore my comments here.

Reply:
Yes, I've used from what is in the previous tests.